standalone-node/installation_scripts/bootable-usb-prepare.sh

```diff
@@ -110,6 +110,16 @@ if [ -z "$host_type" ] || { [ "$host_type" != "kubernetes" ] && [ "$host_type" !
     exit 1
 fi
 
+# Ask the password from cmd line 
+echo -n "Please Set the Password for $user_name: "
+stty -echo
+read password
+stty echo
+echo 
+
+# Store the password in hiden file and delete onces it copied to USB
+echo "passwd=$password" > "$(pwd)/.psswd"
```

Comment:
Consider using more restrictive permissions (e.g., chmod 600) on the .psswd file immediately after creation.